xxhash build failure

**Host Environment**
- OS: Windows 10
- Compiler: clang-cl

**To Reproduce**
Try to build a CMake project with xxhash.

**Failure logs** 
```
CMake Error at build/vcpkg_installed/x64-windows/share/xxhash/xxHashTargets.cmake:76 (message):
The imported target "xxHash::xxhsum" references the file 
"C:/Users/path_redacted/build/vcpkg_installed/x64-windows/tools/xxhash/xxhsum.exe"
but this file does not exist.  Possible reasons include:
```

**Additional context**
The following xxhash defines were added:
```
add_compile_definitions(XXH_STATIC_LINKING_ONLY)
add_compile_definitions(XXH_INLINE_ALL)
```

I have tried removing these definitions and rebuilding to no effect.

xxhash was included as follows:
```
find_package(xxHash CONFIG REQUIRED)
target_link_libraries(main PRIVATE xxHash::xxHash)
```

Please let me know if I can provide any additional information.

--- a/ports/xxhash/CONTROL
+++ b/ports/xxhash/CONTROL
@@ -1,4 +1,8 @@
 Source: xxhash
 Version: 0.8.0
+Port-Version: 1
 Homepage: https://github.com/Cyan4973/xxHash
 Description: Extremely fast hash algorithm
+
+Feature: xxhsum
+Description: Build the xxhsum binary

--- a/ports/xxhash/portfile.cmake
+++ b/ports/xxhash/portfile.cmake
@@ -1,5 +1,3 @@
-vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
-
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO Cyan4973/xxHash
@@ -8,16 +6,24 @@ vcpkg_from_github(
     HEAD_REF dev
 )
 
+vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
+    xxhsum XXHASH_BUILD_XXHSUM
+)
+
 vcpkg_configure_cmake(
     SOURCE_PATH ${SOURCE_PATH}/cmake_unofficial
     PREFER_NINJA
+    OPTIONS ${FEATURE_OPTIONS}
 )
 
 vcpkg_install_cmake()
 vcpkg_copy_pdbs()
 vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/xxHash TARGET_PATH share/${PORT})
 
-file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/bin ${CURRENT_PACKAGES_DIR}/debug/bin)
+if("xxhsum" IN_LIST FEATURES)
+    vcpkg_copy_tools(TOOL_NAMES xxhsum AUTO_CLEAN)
+endif()
+
 file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/share)
 
 file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include)

